login-workflow/src/components/password-strength-checker/password-strength-checker.component.scss

```diff
@@ -0,0 +1,20 @@
+.pxb-auth-password-check {
+    height: auto;
+    ::ng-deep .mat-list-base .mat-list-item .mat-list-item-content {
+        padding: 0;
+    }
+    .validation-message {
+        color: #424e54; //black[500]
```

Comment:
isn't this the default text color?

Reply:
Yes this is correct; I think this can be removed.